application/templates/_shared/_contains_corrections.html

```diff
@@ -0,0 +1,3 @@
+<div class="govuk-inset-text">
+  This page corrects mistakes in a previous version. <a class="govuk-link" href="#full-page-history">See details</a>.
```

Comment:
Kinda worried that we could break this link at some point and not notice.

Reply:
Added be6580f7c2bdfe3d9f23790e2e5f8334ca160ead to hopefully catch this.